examples/toast.html

```diff
@@ -160,10 +160,20 @@ <h1>Open Window Handler</h1>
                 // Note: This manual check for query string is for backwards compatibility of this
                 // deployed example.  In your own apps, you can assume the query param will be
                 // provided by Genesys Cloud if you have configured it in your app's config.
-                let envQueryParamName = 'pcEnvironment';
-                if (window && window.location && typeof window.location.search === 'string' &&
-                    window.location.search.indexOf(envQueryParamName) >= 0) {
-                    myClientApp = new window.purecloud.apps.ClientApp({pcEnvironmentQueryParam: envQueryParamName});
+                const envQueryParamName = 'pcEnvironment';
+                const hostQueryParamName = 'gcHostOrigin';
+                const targetEnvQueryParamName = 'gcTargetEnv';
+                const locationSearch = (window && window.location && typeof window.location.search === 'string') ? window.location.search : '';
```

Comment:
just a heads up/justification ...

a long time ago, when we started this repo, we still had to support IE 9/10.  So, we also tried to keep the examples es5 compat in source.

So, if you see lets or overly protective checks, that's the historical justification.  Def fine to use modern features and assume we're running on no-less than Chromium Edge these days.  Safari is our new IE though :joy:

If you're curious, here are our [supported browsers](https://help.mypurecloud.com/articles/genesys-cloud-requirements/)